apps/site/components/ExtendableQueryTable/ExtendableQueryTable.tsx

```diff
@@ -0,0 +1,32 @@
+import { ReactNode } from "react"
+
+export type ExtendableQueryTableProps = {
+    children: ReactNode
+}
+
+const ExtendableQueryTable = ({ children }: ExtendableQueryTableProps) => {
```

Comment:
Since `ExtendableQueryTableProps` only has children as a prop, you can directly use react's `PropsWithChildren` type.

```suggestion
const ExtendableQueryTable = ({ children }: PropsWithChildren) => {
```

Reply:
done [here](https://github.com/vtex/faststore/pull/1942/commits/59f909d3f32fb15692f41e90dde311f9c74adc4c)